roles/cds_for_icpc/files/deploy_backup_tar

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+from os import listdir, chdir, mkdir
+from os.path import isfile, islink, getmtime, join, realpath, isdir
+import tarfile
+
+backup_file = "backups.tar"
```

Comment:
If you make an ansible template out of this file you can directly pass the value for this.

Reply:
It is a constant,  I have changed it to an uppercase variable, and added documentation.